src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -740,6 +739,11 @@ internal override byte[] GetLocalsSignature()
 
         internal override unsafe void GetEHInfo(int excNumber, void* exc)
         {
+            if (m_exceptions is null)
```

Comment:
The VM side should be calling this method only when `m_exceptions  != null`. Assert may be better.

Reply:
Ok.